docs/tests/integration/test_wazuh_db/test_wazuh_db.md

```diff
@@ -0,0 +1,61 @@
+# Test Wazuh DB
+
+All these tests are meant to send a specific command to the socket (valid or not) and compare the resulting output with the expected result.
```

Comment:
Y suggest moving `(valid or not)` before to `to the socket`. In the current location seems like `the socket` could be valid or not, and I think that you are referring to the `command`.

Reply:
It's true, fixed!